src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table (name = "GROUP")
```

Comment:
As Andrzej mentioned, change to "GROUPS" so that will not interfere with SQL reserved words.

Reply:
DONE :)